contracts/modules/STO/STOStorage.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity ^0.5.0;
  */
 
 contract STOStorage {
+    bytes32 internal constant INVESTORFLAGS = "INVESTORFLAGS";
```

Comment:
Here we can also add the hex of the string

Reply:
The key is not sha3("INVESTORFLAGS") but sha3("INVESTORFLAGS" + $Investor_address) therefore, we can't pre-compute it.